events/wc_druidic_events.txt

```diff
@@ -23,13 +25,25 @@ character_event = {
 	trigger = {
 		fury_of_wolf_human_event_trigger = yes
 		
-		NOT = { has_global_flag = ongoing_fury_of_wolf_chain_flag }
-		
-		is_primary_war_defender = yes
+		AND = { 
+			NOT = { has_global_flag = ongoing_fury_of_wolf_chain_flag }
+			OR = { 
+				any_courtier_or_vassal = { has_artifact = book_of_ur }
+				NOT = { has_dlc = "Mystics" }
+			}
+			OR = { 
+				is_primary_war_defender = yes
+				any_neighbor_independent_ruler = { culture = scourge }
```

Comment:
There may be Holy Light lich with Scourge culture.

Reply:
Changed to use death_god religion instead.